Updated the GitLab installation docs (#339)
port-labs/port-docs: docs/build-your-software-catalog/sync-data-to-catalog/git/gitlab/installation.md

   - `GROUPS_TO_REPOS` - A Projects to group mapping, refer to the tip section below for a syntax example showing how to format this script parameter

>    - `SKIP_WEBHOOK_CREATION` - This option allows you to skip the webhook creation process and get only the current entities in your GitLab.

Reviewer: ```suggestion
   - `SKIP_WEBHOOK_CREATION` - This option allows you to skip the webhook creation process and simply ingest the current entities from your GitLab to Port.
```

   :::note

>    In case `SKIP_WEBHOOK_CREATION` option is set to `true` - the `personal access token` can include `read_api` scope only

Reviewer: ```suggestion
   In case `SKIP_WEBHOOK_CREATION` option is set to `true` - the GitLab `personal access token` does not require the `api` scope, and can instead be limited to the `read_api` scope.
```
